package.json

```diff
@@ -23,6 +23,7 @@
     "start:dev": "npm run build && NODE_ENV=development npm run watch",
     "start:test": "export $(cat test.env) && npm run start",
     "test": "jest --runInBand",
+    "test-debug": "npm test --watch -t",
```

Comment:
was this intended to be left in?

Reply:
@rgforsyth It is intended as we felt it will be beneficial later for debugging.